core/db/schema.sql

```diff
@@ -50,33 +82,42 @@ CREATE TABLE messages (
 );
 
 
+--
+-- Name: messages_message_id_seq; Type: SEQUENCE; Schema: public; Owner: -
+--
+
+CREATE SEQUENCE messages_message_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
 
+--
+-- Name: messages_message_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: -
+--
 
+ALTER SEQUENCE messages_message_id_seq OWNED BY messages.message_id;
 
--- Table: tasks
 
--- DROP TABLE tasks;
+--
+-- Name: tasks; Type: TABLE; Schema: public; Owner: -; Tablespace: 
+--
 
-CREATE TABLE tasks
-(
-  task_id serial NOT NULL,
-  task character varying(80) NOT NULL,
-  username character varying(50) NOT NULL,
-  posted date DEFAULT ('now'::text)::date,
-  --Add a numeric sort
-  CONSTRAINT tasks_pkey PRIMARY KEY (task_id)
```

Comment:
Oh, whoops. I just did a pg_dump. I'll  fix that.
On Feb 2, 2015 5:36 PM, "trevren11" <notifications@github.com> wrote:

> In core/db/schema.sql
> <https://github.com/BYU-ODH/hlrdesk/pull/123#discussion_r23973215>:
>
> >
> > -CREATE TABLE tasks
> > -(
> > -  task_id serial NOT NULL,
> > -  task character varying(80) NOT NULL,
> > -  username character varying(50) NOT NULL,
> > -  posted date DEFAULT ('now'::text)::date,
> > -  --Add a numeric sort
> > -  CONSTRAINT tasks_pkey PRIMARY KEY (task_id)
>
> You are removing my constraint and note to self, how dare you!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BYU-ODH/hlrdesk/pull/123/files#r23973215>.
>